packages/grafana-ui/src/components/uPlot/utils.test.ts

```diff
@@ -1,7 +1,7 @@
 import { FieldMatcherID, fieldMatchers, FieldType, MutableDataFrame } from '@grafana/data';
 import { BarAlignment, GraphDrawStyle, GraphTransform, LineInterpolation, StackingMode } from '@grafana/schema';
 
-import { preparePlotFrame } from '../../../../../public/app/core/components/GraphNG/utils';
+import { preparePlotFrame } from '..';
```

Comment:
I'm assuming this was checked with data viz team, but this export says
```
// Graveyard: exported, but no longer used internally
// These will be removed in the future
```

Reply:
naw i haven't checked, but i'll wait for a review from them 👍